pkg/integration/components/viewDriver.go

```diff
@@ -50,6 +51,74 @@ func (self *ViewDriver) Lines(matchers ...*matcher) *ViewDriver {
 	return self.assertLines(matchers...)
 }
 
+func (self *ViewDriver) getSelectedLines() ([]string, error) {
+	if self.getSelectedLinesFn == nil {
+		view := self.t.gui.View(self.getView().Name())
+
+		return []string{view.SelectedLine()}, nil
+	}
+
+	return self.getSelectedLinesFn()
+}
+
+func (self *ViewDriver) SelectedLines(matchers ...*matcher) *ViewDriver {
+	self.t.assertWithRetries(func() (bool, string) {
+		selectedLines, err := self.getSelectedLines()
+		if err != nil {
+			return false, err.Error()
+		}
+
+		selectedContent := strings.Join(selectedLines, "\n")
+		expectedContent := expectedContentFromMatchers(matchers)
+
+		if len(selectedLines) != len(matchers) {
+			return false, fmt.Sprintf("Expected the following to be selected:\n-----\n%s\n-----\nBut got:\n-----\n%s\n-----", expectedContent, selectedContent)
+		}
+
+		for i, line := range selectedLines {
+			ok, message := matchers[i].test(line)
+			if !ok {
+				return false, fmt.Sprintf("Error: %s. Expected the following to be selected:\n-----\n%s\n-----\nBut got:\n-----\n%s\n-----", message, expectedContent, selectedContent)
+			}
+		}
+
+		return true, ""
+	})
+
+	return self
+}
+
+func (self *ViewDriver) ContainsLines(matchers ...*matcher) *ViewDriver {
```

Comment:
Now we have Lines, TopLines, ContainsLines, and SelectedLines. We'll need to update the naming so they're all consistent e.g. AllLines, TopLines, SomeLines, and SelectedLines. Something like that. I'll do that in another PR